<?xml version="1.0" encoding="utf-8"?>
<xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="1.2" xsi:schemaLocation="urn:oasis:names:tc:xliff:document:1.2 xliff-core-1.2-transitional.xsd">
  <file datatype="xml" original="DriveNotFoundException.xml" source-language="en-US" target-language="it-IT">
    <header>
      <tool tool-id="mdxliff" tool-name="mdxliff" tool-version="1.0-15c36f0" tool-company="Microsoft" />
      <xliffext:skl_file_name xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">02cd5861-7ce2-4a82-b358-31f8435a0ac5e1b38525694a6fdaf0b9dd694074396b31b89adf.skl</xliffext:skl_file_name>
      <xliffext:version xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">1.2</xliffext:version>
      <xliffext:ms.openlocfilehash xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">e1b38525694a6fdaf0b9dd694074396b31b89adf</xliffext:ms.openlocfilehash>
      <xliffext:ms.sourcegitcommit xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">d31dc2ede16f6f7bc64e90d9f897ff54c4e3869b</xliffext:ms.sourcegitcommit>
      <xliffext:ms.lasthandoff xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">04/03/2018</xliffext:ms.lasthandoff>
      <xliffext:moniker_ids xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">netcore-1.0,netcore-1.1,netcore-2.0,netcore-2.1,netframework-4.5.1,netframework-4.5.2,netframework-4.5,netframework-4.6.1,netframework-4.6.2,netframework-4.6,netframework-4.7.1,netframework-4.7,netstandard-2.0,xamarinandroid-7.1,xamarinios-10.8,xamarinmac-3.0</xliffext:moniker_ids>
    </header>
    <body>
      <group id="content" extype="content">
        <trans-unit id="101" translate="yes" xml:space="preserve" uid="T:System.IO.DriveNotFoundException">
          <source>The exception that is thrown when trying to access a drive or share that is not available.</source>
          <target state="translated">Eccezione generata durante il tentativo di accesso a un'unità o condivisione non disponibile.</target>       </trans-unit>
        <trans-unit id="102" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.IO.DriveNotFoundException">
          <source>The <ph id="ph1">&lt;xref:System.IO.DriveNotFoundException&gt;</ph> class uses the HRESULT COR_E_DIRECTORYNOTFOUND.</source>
          <target state="translated">La <ph id="ph1">&lt;xref:System.IO.DriveNotFoundException&gt;</ph> classe utilizza HRESULT COR_E_DIRECTORYNOTFOUND.</target>       </trans-unit>
        <trans-unit id="103" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.IO.DriveNotFoundException">
          <source>This exception can be thrown when a Win32 ERROR_PATH_NOT_FOUND HRESULT error is encountered.</source>
          <target state="translated">Questa eccezione può essere generata quando viene rilevato un errore di Win32 ERROR_PATH_NOT_FOUND HRESULT.</target>       </trans-unit>
        <trans-unit id="104" translate="yes" xml:space="preserve" uid="T:System.IO.DriveNotFoundException">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.IO.DriveNotFoundException" /&gt;</ph> class.</source>
          <target state="translated">Inizializza una nuova istanza della classe <ph id="ph1">&lt;see cref="T:System.IO.DriveNotFoundException" /&gt;</ph>.</target>       </trans-unit>
        <trans-unit id="105" translate="yes" xml:space="preserve" uid="M:System.IO.DriveNotFoundException.#ctor">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.IO.DriveNotFoundException" /&gt;</ph> class with its message string set to a system-supplied message and its HRESULT set to COR_E_DIRECTORYNOTFOUND.</source>
          <target state="translated">Inizializza una nuova istanza della classe <ph id="ph1">&lt;see cref="T:System.IO.DriveNotFoundException" /&gt;</ph> con la relativa stringa di messaggio impostata su un messaggio fornito dal sistema e il relativo HRESULT impostato su COR_E_DIRECTORYNOTFOUND.</target>       </trans-unit>
        <trans-unit id="106" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.IO.DriveNotFoundException.#ctor">
          <source>This constructor initializes the <ph id="ph1">&lt;xref:System.Exception.Message%2A?displayProperty=nameWithType&gt;</ph> property of the new instance to a system-supplied message that describes the error, such as "Could not find the specified directory."</source>
          <target state="translated">Questo costruttore inizializza la <ph id="ph1">&lt;xref:System.Exception.Message%2A?displayProperty=nameWithType&gt;</ph> proprietà della nuova istanza di un messaggio fornito dal sistema che descrive l'errore, ad esempio "Impossibile trovare la directory specificata."</target>       </trans-unit>
        <trans-unit id="107" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.IO.DriveNotFoundException.#ctor">
          <source>This message is localized based on the current system culture.</source>
          <target state="translated">Questo messaggio è localizzato in base alle impostazioni cultura correnti del sistema.</target>       </trans-unit>
        <trans-unit id="108" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.IO.DriveNotFoundException.#ctor">
          <source>The <ph id="ph1">&lt;xref:System.Exception.InnerException%2A?displayProperty=nameWithType&gt;</ph> property of the new instance is initialized to <ph id="ph2">`null`</ph>.</source>
          <target state="translated">Il <ph id="ph1">&lt;xref:System.Exception.InnerException%2A?displayProperty=nameWithType&gt;</ph> proprietà della nuova istanza viene inizializzata su <ph id="ph2">`null`</ph>.</target>       </trans-unit>
        <trans-unit id="109" translate="yes" xml:space="preserve" uid="M:System.IO.DriveNotFoundException.#ctor(System.String)">
          <source>A <ph id="ph1">&lt;see cref="T:System.String" /&gt;</ph> object that describes the error.</source>
          <target state="translated">Oggetto <ph id="ph1">&lt;see cref="T:System.String" /&gt;</ph> che descrive l'errore.</target>       </trans-unit>
        <trans-unit id="110" translate="yes" xml:space="preserve" uid="M:System.IO.DriveNotFoundException.#ctor(System.String)">
          <source>The caller of this constructor is required to ensure that this string has been localized for the current system culture.</source>
          <target state="translated">Il chiamante del costruttore deve garantire che la stringa sia stata localizzata in base alle impostazioni cultura correnti del sistema.</target>       </trans-unit>
        <trans-unit id="111" translate="yes" xml:space="preserve" uid="M:System.IO.DriveNotFoundException.#ctor(System.String)">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.IO.DriveNotFoundException" /&gt;</ph> class with the specified message string and the HRESULT set to COR_E_DIRECTORYNOTFOUND.</source>
          <target state="translated">Inizializza una nuova istanza della classe <ph id="ph1">&lt;see cref="T:System.IO.DriveNotFoundException" /&gt;</ph> con la stringa di messaggio specificata e HRESULT impostato su COR_E_DIRECTORYNOTFOUND.</target>       </trans-unit>
        <trans-unit id="112" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.IO.DriveNotFoundException.#ctor(System.String)">
          <source>This constructor initializes the <ph id="ph1">&lt;xref:System.Exception.Message%2A?displayProperty=nameWithType&gt;</ph> property of the new instance using the <ph id="ph2">`message`</ph> parameter.</source>
          <target state="translated">Questo costruttore inizializza la proprietà <ph id="ph1">&lt;xref:System.Exception.Message%2A?displayProperty=nameWithType&gt;</ph> della nuova istanza utilizzando il parametro <ph id="ph2">`message`</ph>.</target>       </trans-unit>
        <trans-unit id="113" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.IO.DriveNotFoundException.#ctor(System.String)">
          <source>The <ph id="ph1">&lt;xref:System.Exception.InnerException%2A?displayProperty=nameWithType&gt;</ph> property of the new instance is initialized to <ph id="ph2">`null`</ph>.</source>
          <target state="translated">Il <ph id="ph1">&lt;xref:System.Exception.InnerException%2A?displayProperty=nameWithType&gt;</ph> proprietà della nuova istanza viene inizializzata su <ph id="ph2">`null`</ph>.</target>       </trans-unit>
        <trans-unit id="114" translate="yes" xml:space="preserve" uid="M:System.IO.DriveNotFoundException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>A <ph id="ph1">&lt;see cref="T:System.Runtime.Serialization.SerializationInfo" /&gt;</ph> object that contains the serialized object data about the exception being thrown.</source>
          <target state="translated">Oggetto <ph id="ph1">&lt;see cref="T:System.Runtime.Serialization.SerializationInfo" /&gt;</ph> contenente i dati serializzati dell'oggetto relativi all'eccezione generata.</target>       </trans-unit>
        <trans-unit id="115" translate="yes" xml:space="preserve" uid="M:System.IO.DriveNotFoundException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>A <ph id="ph1">&lt;see cref="T:System.Runtime.Serialization.StreamingContext" /&gt;</ph> object that contains contextual information about the source or destination of the exception being thrown.</source>
          <target state="translated">Oggetto <ph id="ph1">&lt;see cref="T:System.Runtime.Serialization.StreamingContext" /&gt;</ph> contenente le informazioni contestuali relative all'origine o alla destinazione dell'eccezione generata.</target>       </trans-unit>
        <trans-unit id="116" translate="yes" xml:space="preserve" uid="M:System.IO.DriveNotFoundException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.IO.DriveNotFoundException" /&gt;</ph> class with the specified serialization and context information.</source>
          <target state="translated">Inizializza una nuova istanza della classe <ph id="ph1">&lt;see cref="T:System.IO.DriveNotFoundException" /&gt;</ph> con le informazioni di serializzazione e di contesto specificate.</target>       </trans-unit>
        <trans-unit id="117" translate="yes" xml:space="preserve" uid="M:System.IO.DriveNotFoundException.#ctor(System.String,System.Exception)">
          <source>The error message that explains the reason for the exception.</source>
          <target state="translated">Messaggio di errore in cui viene indicato il motivo dell'eccezione.</target>       </trans-unit>
        <trans-unit id="118" translate="yes" xml:space="preserve" uid="M:System.IO.DriveNotFoundException.#ctor(System.String,System.Exception)">
          <source>The exception that is the cause of the current exception.</source>
          <target state="translated">Eccezione che ha determinato l'eccezione corrente.</target>       </trans-unit>
        <trans-unit id="119" translate="yes" xml:space="preserve" uid="M:System.IO.DriveNotFoundException.#ctor(System.String,System.Exception)">
          <source>If the <bpt id="p1">&lt;c&gt;</bpt>innerException<ept id="p1">&lt;/c&gt;</ept> parameter is not <ph id="ph1">&lt;see langword="null" /&gt;</ph>, the current exception is raised in a <ph id="ph2">&lt;see langword="catch" /&gt;</ph> block that handles the inner exception.</source>
          <target state="translated">Se il parametro <bpt id="p1">&lt;c&gt;</bpt>innerException<ept id="p1">&lt;/c&gt;</ept> non è <ph id="ph1">&lt;see langword="null" /&gt;</ph>, l'eccezione corrente viene generata in un blocco <ph id="ph2">&lt;see langword="catch" /&gt;</ph> in cui viene gestita l'eccezione interna.</target>       </trans-unit>
        <trans-unit id="120" translate="yes" xml:space="preserve" uid="M:System.IO.DriveNotFoundException.#ctor(System.String,System.Exception)">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.IO.DriveNotFoundException" /&gt;</ph> class with the specified error message and a reference to the inner exception that is the cause of this exception.</source>
          <target state="translated">Inizializza una nuova istanza della classe <ph id="ph1">&lt;see cref="T:System.IO.DriveNotFoundException" /&gt;</ph> con il messaggio di errore specificato e un riferimento all'eccezione interna che è la causa dell'eccezione.</target>       </trans-unit>
        <trans-unit id="121" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.IO.DriveNotFoundException.#ctor(System.String,System.Exception)">
          <source>An exception that is thrown as a direct result of a previous exception should include a reference to the previous exception in the <ph id="ph1">&lt;xref:System.Exception.InnerException%2A&gt;</ph> property.</source>
          <target state="translated">Un'eccezione generata come risultato diretto di un'eccezione precedente deve includere un riferimento all'eccezione precedente nella proprietà <ph id="ph1">&lt;xref:System.Exception.InnerException%2A&gt;</ph>.</target>       </trans-unit>
        <trans-unit id="122" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.IO.DriveNotFoundException.#ctor(System.String,System.Exception)">
          <source>The <ph id="ph1">&lt;xref:System.Exception.InnerException%2A&gt;</ph> property returns the same value that is passed into the constructor, or <ph id="ph2">`null`</ph> if the <ph id="ph3">&lt;xref:System.Exception.InnerException%2A&gt;</ph> property does not supply the inner exception value to the constructor.</source>
          <target state="translated">La proprietà <ph id="ph1">&lt;xref:System.Exception.InnerException%2A&gt;</ph> restituisce lo stesso valore passato al costruttore oppure <ph id="ph2">`null`</ph> nel caso in cui la proprietà <ph id="ph3">&lt;xref:System.Exception.InnerException%2A&gt;</ph> non fornisca il valore di eccezione interna al costruttore.</target>       </trans-unit>
        <trans-unit id="123" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.IO.DriveNotFoundException.#ctor(System.String,System.Exception)">
          <source>The following table shows the initial property values for an instance of <ph id="ph1">&lt;xref:System.IO.DirectoryNotFoundException&gt;</ph>.</source>
          <target state="translated">Nella tabella seguente vengono illustrati i valori di proprietà iniziali per un'istanza di <ph id="ph1">&lt;xref:System.IO.DirectoryNotFoundException&gt;</ph>.</target>       </trans-unit>
        <trans-unit id="124" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.IO.DriveNotFoundException.#ctor(System.String,System.Exception)">
          <source>Property</source>
          <target state="translated">Proprietà</target>       </trans-unit>
        <trans-unit id="125" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.IO.DriveNotFoundException.#ctor(System.String,System.Exception)">
          <source>Value</source>
          <target state="translated">Valore</target>       </trans-unit>
        <trans-unit id="126" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.IO.DriveNotFoundException.#ctor(System.String,System.Exception)">
          <source>The inner exception reference.</source>
          <target state="translated">Riferimento all'eccezione interna.</target>       </trans-unit>
        <trans-unit id="127" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.IO.DriveNotFoundException.#ctor(System.String,System.Exception)">
          <source>The error message string.</source>
          <target state="translated">Stringa del messaggio di errore.</target>       </trans-unit>
      </group>
    </body>
  </file>
</xliff>